requirements/dev.txt

```diff
@@ -126,13 +126,13 @@ pep517==0.13.0
     # via
     #   -r requirements/pip-tools.txt
     #   build
-pillow==9.2.0
+pillow==9.3.0
```

Comment:
No breaking change mentioned in the new version: https://github.com/python-pillow/Pillow/blob/main/CHANGES.rst#930-2022-10-29